samples/sensor/max31865/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
```

Comment:
Please move the sample changes to a separate commit in this PR. Please also add a `sample.yaml`